.github/CODEOWNERS

```diff
@@ -0,0 +1 @@
+* @dstufft @ewdurbin @di
```

Comment:
```suggestion
* @pypi/warehouse-committers
```

Reply:
I _think_ this will have the effect of sending email notifications to everyone in there that a review has been requested for every PR, maybe we want `@pypi/warehouse-admins` instead?